src/en/koharu/src/eu/kanade/tachiyomi/extension/en/koharu/KoharuDto.kt

```diff
@@ -0,0 +1,75 @@
+package eu.kanade.tachiyomi.extension.en.koharu
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+class Tag(
+    var name: String,
+    var namespace: Int = 0,
+)
+
+@Serializable
+class Books(
+    val entries: List<Entry> = emptyList(),
+    val total: Int = 0,
+    val limit: Int = 0,
+    val page: Int,
+)
+
+@Serializable
+class Entry(
+    val id: Int,
+    val public_key: String,
+    val title: String,
+    val thumbnail: Thumbnail,
+)
+
+@Serializable
+class MangaEntry(
+    val id: Int,
+    val title: String,
+    val public_key: String,
+    val created_at: Long = 0L,
+    val updated_at: Long?,
+    val thumbnails: Thumbnails,
+    val tags: List<Tag> = emptyList(),
+    val data: Data,
+)
+
+@Serializable
+class Thumbnails(
+    val base: String,
+    val main: Thumbnail,
+    val entries: List<Thumbnail>,
+)
+
+@Serializable
+class Thumbnail(
+    val path: String,
+)
+
+@Serializable
+class Data(
+    val `0`: DataKey,
+    val `780`: DataKey = `0`,
+    val `980`: DataKey = `780`,
+    val `1280`: DataKey = `980`,
+    val `1600`: DataKey = `1280`,

```

Comment:
why assign one previous value as default?

Reply:
Their API sometimes doesn't have a specific resolution, like 1600x, so I set the previous value as the default to be safe.

Reply:
Hello, sorry for late commit. Can you please check the new code and see if it matches what you asked for?

Reply:
I meant just changing the default assignments order, but I suppose this works too